cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/requestrate/RequestRateConfiguration.java

```diff
@@ -13,11 +16,23 @@
 public class RequestRateConfiguration {
 
     /**
+     * Default is to report all requests.
+     */
+    public static final String ALL_REQUESTS_TO_REPORT = "ALL";
```

Comment:
I agree that ALL can lead to confusion but when you have in configuration:
```
options:
  requestsToReport:
    - ALL
```

Looks much better then:
```
options:
  requestsToReport:
    - :
```

I would sacrifice a bit complex implementation for better readability in configuration yaml. Also when you force statement type consistency level pairs it is not confusing:
```
options:
  requestsToReport:
    - SELECT:ALL
    - UPDATE:ONE
```

Downside is that you need to add all consistency levels you want to report but being explicit is not bad thing IMO.